conf/application.conf

```diff
@@ -185,7 +185,7 @@ pagination {
   }
 
   departure {
-    numberOfMovements = 20
+    numberOfMovements = 2
```

Comment:
This , according the ticket should be 20, we had it in arrivals as 2 just for testing in the UI manually.

Reply:
I changed it to match the arrivals one, but happy to set back to 20